.github/workflows/ci-testing-deploy.yml

```diff
@@ -1732,6 +1732,13 @@ jobs:
         run: ./ci/github/system-testing/swarm-deploy.bash install
       - name: test
         run: ./ci/github/system-testing/swarm-deploy.bash test
+      - name: dump services setting schemas
+        run: export DOCKER_REGISTRY=local; export DOCKER_IMAGE_TAG=production; make settings-schema.json
+      - name: upload services settings schemas
+        uses: actions/upload-artifact@v3
```

Comment:
Could you maybe provide some notes on the lifetime of these artefacts. What is it's retention period?

I would like to avoid the scenario where OPS (@mrnicegyu11 @Surfict) will implement something relying on this and after 1 month their script will no longer work because the artifacts were removed automatically.

I think that if this were boundless with the release this risk should not be there.

Reply:
- Lifetime: "... By default, GitHub stores build logs and artifacts for 90 days, and this retention period can be customized ..." (see [Storing workflow data as artifacts](https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts)
- Note that these artifacts are very easily reproduced by pulling the service image and calling ``settings`` subcommand with ``--as-json-schema``. 
- Yes, i agree